src/js/components/AddCourseSessionForm.js

```diff
@@ -0,0 +1,176 @@
+import React, { Component }     from 'react';
+import {
+  withRouter,
+}                               from 'react-router-dom';
+import API from "../services/api";
+
+class AddCourseSessionForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      start_time: '',
+      session_duration: 0,
+      video_url: '',
+      thumbnail_image_url: '',
+      course_id: '',
+
+      courseCreated: false
+    }
+
+    this.create_course_session = this.create_course_session.bind(this);
+  }
+
+  render() {
+
+    let styles = {
+      button: {
+        width: '100%'
+      }
+    };
+
+    return (
+      <div>
+
+        {this.renderSuccess()}
+
+        * required
+
+        <div className="m-t-15">
+          *Start Time:
+          <input
+            type="datetime-local"
+            className="form-control"
+            placeholder="Start Time"
+            value={this.state.start_time}
+            onChange={(e) => this.handleTextChange('start_time', e.target.value)}
+          />
+        </div>
+
+        <div className="m-t-15">
+          Duration(in minutes. This is optional and will override the default course duration) :
+          <input
+            type="number"
+            className="form-control"
+            placeholder="duration"
+            value={this.state.session_duration}
+            onChange={(e) => this.handleTextChange('session_duration', e.target.value)}
+          />
+        </div>
+
+        <div className="m-t-15">
+          *Video URL:
+          <input
+            type="string"
+            className="form-control"
+            placeholder="Video Url"
+            value={this.state.video_url}
+            onChange={(e) => this.handleTextChange('video_url', e.target.value)}
+          />
+        </div>
+
+        <div className="m-t-15">
+          *Thumbnail URL:
+          <input
+            type="string"
+            className="form-control"
+            placeholder="Thumbnail Image Url"
+            value={this.state.thumbnail_image_url}
+            onChange={(e) => this.handleTextChange('thumbnail_image_url', e.target.value)}
+          />
+        </div>
+
+        <div className="m-t-15">
+          *Course:
+          <select id="course"
+          defaultValue=""
+                  onChange={(e) => this.handleTextChange('course_id', e.target.value)}>
+            <option value="" disabled> Select Course </option>
+            {
+              this.props.courses.map(function(course) {
+              return <option
+                             value={course.id}>{course.name}</option>
+            })
+            }
+          </select>
+        </div>
+
+        {/*this is to choose a live or on demand video, the migration for adding a live bool is on a separate branch, so add this field back in when you've merged everything*/}

```

Comment:
Leave commented code here. Write text in actual website to indicate that live vs. on demand is not implemented and to do it in the terminal.